### PR TITLE
Pagination tests for all models.

### PR DESCRIPTION
**Pull request by christin-wednesday**
### Description

Added pagination tests for All models. 

>**Pagination tests**:
> 1. checks whether the paginated query is available for the pluralized model
> 2. checks whether the `pageInfo` and `total` field values are correct.
> 3. checks if cursors are working correctly

Used `faker` to mock data.

add-pagination-tests

### CODE REVIEW

File: server/utils/testUtils/index.js [hunk 1]
```diff
@@ -1,47 +1,17 @@
 import isNil from 'lodash/isNil';
 
-const testDate = new Date();
-const addressesTable = {
```
Comment (MrigWed) [inline server/utils/testUtils/index.js:4]: Are these not used for other tests?

**Review by MrigWed — COMMENTED** [review]
I guess other tests are using the utils you deleted.... Can you check once?

File: server/utils/testUtils/index.js (hunk 1 above)
Comment (christin-wednesday) [reply server/utils/testUtils/index.js:4]: Moved these into a separate file in mockData.

Comment (christin-wednesday) [reply server/utils/testUtils/index.js:4]: Moved all the mocks into a separate file.

**Review by MrigWed — APPROVED** [review]

**Pull request merged**